stac_planet_api/api.py

```diff
@@ -210,10 +210,9 @@ async def get_search(
     if sortby:
         search_request["sortby"] = [
             {
-                "field": sort[1:] if sort[0] in ["-", "+"] else sort,
-                "direction": "desc" if sort[0] == "-" else "asc",
+                "field": sortby[1:] if sortby[0] in ["-", "+"] else sortby,
+                "direction": "desc" if sortby[0] == "-" else "asc",
             }
-            for sort in sortby
```

Comment:
STAC does allow you to use a comma separated list for sort so you may want to add the for loop back with a `.split(",")`. So it would be:
```
search_request["sortby"] = [
    {
        "field": sort[1:] if sort[0] in ["-", "+"] else sort,
        "direction": "desc" if sort[0] == "-" else "asc",
    }
    for sort in sortby.split(",")
]
```
But the planet API only allows you to sort on one field so the rest looks right.